src/runtime_src/core/edge/drm/Makefile

```diff
@@ -6,6 +6,12 @@ KBUILD_OPTS := M=$(CURDIR)/$(DRIVER)
 KBUILD_OPTS += ARCH=$(TARGET)
 KBUILD_OPTS += CROSS_COMPILE=$(CROSS_COMPILE)
 
+GIT_HASH := `git -C /proj/rdi-xsj/staff/sarabjee/xrt_git/sarab/XRT/src rev-parse --verify HEAD`
```

Comment:
@sarab96 , Can you please explain why hardcoding the path here? 
And xrt-recipe in petalinux doesnt use this make file. It uses the makefile which is inside zocl.
https://gitenterprise.xilinx.com/Yocto/meta-xilinx/blob/2020/meta-xilinx-bsp/recipes-xrt/zocl/zocl_git.bb